server/lorax_server/utils/paged_attention.py

```diff
@@ -33,17 +36,22 @@ def reshape_and_cache(
     key_cache: torch.Tensor,
     value_cache: torch.Tensor,
     slots: torch.Tensor,
+    k_scale: float = 1.0,
+    v_scale: float = 1.0,
 ):
     if FLASH_INFER:
+        if key_cache.dtype == torch.float8_e4m3fn and value_cache.dtype == torch.float8_e4m3fn:
```

Comment:
@tgaddair This is a potential bottleneck, so I am trying to search for a fused kernel.